spec/factories/banners.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :banner do
+    message "MyString"
+    expires_at "2018-07-17 17:30:39"
```

Comment:
Might be a good idea to default to an expiration date in the future to avoid surprises, for example using `Time.now + 1.day` or something.

Reply:
Fixed.